code/API_definitions/network_access_management.yaml

```diff
@@ -132,7 +132,7 @@ info:
     may be used.  If an end-user replaces the equipment, all existing isolated networks SHOULD be synced to the new
     device representing this mesh.  Alternatively, a synthetic hardware address may also be used if the specification
     for the address type allows.
-  version: wip
+  version: 0.1.0

```

Comment:
Leave this as wip until the commonalities and ICM changes are finalized and we've incorporated them.

Reply:
> Leave this as wip until the commonalities and ICM changes are finalized and we've incorporated them.

Actually it is correct that the "release PR" will be the one setting the version which will be released. The important point is that all other PRs (e.g. the once addressing #18 and #19) must be merged before the "release PR".
BTW: the PR to create the release should change if possible only the version field and the base path, but not be used to address issues etc.